.travis.yml

```diff
@@ -39,9 +59,15 @@ before_install:
 
 install:
     - ./autogen.sh
+    # Build rdma-core only in linux as OS X doesn't have verbs support
+    - if [[ "$TRAVIS_OS_NAME" == "linux" ]] ; then
+        RDMA_CORE_BRANCH=v20 ;
+        git clone --depth 1 -b $RDMA_CORE_BRANCH https://github.com/linux-rdma/rdma-core.git && cd rdma-core && bash build.sh && cd - ;
```

Comment:
It does not. I've attempted to use some of the other scripts under rdma-core/buildlib to perform an install, but it isn't as straight forward as I'd hope. 

My guess is that those scripts are highly specific to the rdma-core travis environment. I've attempted to use a few of them without success.

Reply:
You might need to figure out the Right magic to get rdma-core installed into a temporary tree somewhere, and then specify `--with-verbs=/path/to/that/temp/tree` in Libfabric's `configure` invocation.

Reply:
I've switched to using --enable-verbs. It turns out that the problem that I was observing was run-time resolution of the library dependencies of libfabric. ldd against libfabric would easily show that the ibverbs/rdmacm libraries could not be found and that is because they existed in a location outside of what was known from the ld cache. 

The solution that I'm proposing here is to specify the runtime path of those custom ibverbs/rdma-core libraries as part of the linking stage of libfabric so that dependent applications/libfabric can find librdmacm and libibverbs without specifying LD_LIBRARY_PATH at make time.